### ISSUE TITLE
Exception with push on rebuild

### ISSUE THREAD

**Issue by Dudeplayz** [mcollovati/vertx-vaadin#39]
Hi,
I recently enabled push. Now I discovered the behavior, that an exception is thrown if I rebuild the app while I have the browser window opened:
```
SCHWERWIEGEND: Unhandled exception
java.lang.NoSuchMethodError: 'java.lang.String com.vaadin.flow.server.VaadinService.createSessionExpiredJSON()'
	at com.github.mcollovati.vertx.vaadin.sockjs.communication.SockJSPushHandler.callWithUi(SockJSPushHandler.java:237)
	at com.github.mcollovati.vertx.vaadin.sockjs.communication.SockJSPushHandler.lambda$onConnect$3(SockJSPushHandler.java:183)
	at com.github.mcollovati.vertx.vaadin.sockjs.communication.SockJSRoutingContext.lambda$next$1(SockJSPushHandler.java:587)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:510)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:518)
	at io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1044)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
vertx: `3.8.5`
vaadin: `14.2.0.beta1`
vertx-vaadin: `14.1.2-SNAPSHOT`

IDE is marking following:
![image](https://user-images.githubusercontent.com/15174076/81018939-d6836d80-8e65-11ea-83ad-f7e9e73f2d55.png)

**Comment by mcollovati** [comment]
Thanks for reporting.
It seems there are some breaking changes on vaadin 14.2; for example `VaadinService.createSessionExpiredJSON` now takes a boolean parameter.

I think I should open a new dedicated branch; I'll work on this in the next days.

**Comment by mcollovati** [comment]
Published 14.2.0-SNAPSHOT as a first snapshot for Vaadin 14.2.x upgrade

**Comment by Dudeplayz** [comment]
It seems to work. Thanks for the fast fixing 👍.
